src/token.rs

```diff
@@ -76,6 +76,8 @@ pub struct Token {
     storage: Box<dyn Storage>,
     session_objects: HashMap<CK_OBJECT_HANDLE, Object>,
     logged: CK_USER_TYPE,
+    #[cfg(feature = "fips")]
+    kdf_selftest_ran: bool,
```

Comment:
Should this variable name be more specific?

Reply:
we are changing how (all of) this will be handled after a discussion with @Jakuje so wait a bit for reviewing details

Reply:
Yeah. This was mostly PoC. All revamped now.